css/custom.css

```
@@ -668,6 +668,31 @@ body.modal-open .modalcontent {
   margin-right: 10%;
 }
 
+.quiz-right-bottom table {
+  width: 90%;
+  justify-content: center;
```

This line can be deleted unless we add `display: flex;` as well, otherwise it does nothing
```suggestion
  display: flex;
  justify-content: center;
```